library/modules/Textures.cpp

```diff
@@ -322,6 +322,7 @@ struct tracking_stage_new_region : df::viewscreen_new_regionst {
 
     DEFINE_VMETHOD_INTERPOSE(void, logic, ()) {
         if (this->m_raw_load_stage != this->raw_load_stage) {
+            CoreSuspender guard;
```

Comment:
these methods will never be invoked by anything other than DF's simulation thread
the only reason they would need a core lock is if they are going to manipulate structures in DFHack's core
we don't need to arbitrate access to DF's core in these because there is no plausible scenario in which they are called by anything other than the DF simulation thread

Reply:
reverted